src/coreclr/jit/emitarm64.h

```diff
@@ -963,7 +963,7 @@ void emitIns(instruction ins);
 
 void emitIns_I(instruction ins, emitAttr attr, ssize_t imm);
 
-void emitIns_R(instruction ins, emitAttr attr, regNumber reg);
+void emitIns_R(instruction ins, emitAttr attr, regNumber reg, insOpts opt = INS_OPTS_NONE);
```

Comment:
why do we need this change? isn't the `aesimc` and `aesmc` always on `.B`?

Reply:
I guess we don't need `insOpts` since those instructions are implicit on `B` and `S`. Though, we will have to introduce this when we encounter instructions that can have different arrangements.